figures/S9.supplemental_figure9/scripts/S9.r

```diff
@@ -0,0 +1,205 @@
+suppressPackageStartupMessages(suppressWarnings(library(ggplot2))) # plotting
+suppressPackageStartupMessages(suppressWarnings(library(dplyr))) # data manipulation
+suppressPackageStartupMessages(suppressWarnings(library(argparser))) # command line arguments
+suppressPackageStartupMessages(suppressWarnings(library(patchwork))) # plot patchwork
+suppressPackageStartupMessages(suppressWarnings(library(reshape2))) # data manipulation
+suppressPackageStartupMessages(suppressWarnings(library(ggridges))) # ridgeline plots
+suppressPackageStartupMessages(suppressWarnings(library(RColorBrewer))) # color palettes
+suppressPackageStartupMessages(suppressWarnings(library(cowplot))) # ggplot2 drawing
+
+
+source("../../utils/figure_themes.r")
+
+
+model_name <- "MultiClass_MLP"
+cell_type <- "PBMC"
+
+# load in the probabilities
+treatment_holdout_probabilities_path <- file.path(
+    paste0(
+        "../../../4.sc_Morphology_Neural_Network_MLP_Model/results/Multi_Class/",model_name,"/",cell_type,"/probabilities.parquet"
+    )
+)
+# read in the data from the parquet file
+probabilities <- arrow::read_parquet(
+    treatment_holdout_probabilities_path
+)
+head(probabilities,2)
+
+unique(probabilities$data_split)
+unique(probabilities$shuffle)
+
+# # remove rows where shuffle is true
+# probabilities <- probabilities[!grepl("TRUE", probabilities$shuffle),]
```

Comment:
Recommend removing commented out code if not used anymore.

Reply:
removed commented out code